wradlib/io.py

```diff
@@ -846,7 +846,9 @@ def read_RADOLAN_composite(fname, missing=-9999, loaddata=True):
     NODATA = missing
     mask = 0xFFF  # max value integer
 
-    f = get_radolan_filehandle(fname)

```

Comment:
We can try to prevent the explicit type-check like this:

``` python
try:
    header = read_radolan_header(f)
except AttributeError:
    f = get_radolan_filehandle(f)
    header = read_radolan_header(f)
```